sdk/quantum/azure-quantum-jobs/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure.quantum.jobs</groupId>
+  <artifactId>azure-quantum-jobs</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure.quantum.jobs:azure-quantum-jobs;current} -->
+
+  <name>Microsoft Azure client library for Azure Quantum service</name>
+  <description>This package contains the Microsoft Azure Quantum SDK.</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+ <scm>
+    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+    <tag>HEAD</tag>
+  </scm>
+
+   <properties>
+    <!-- quantum jobs temporarily skipping code coverage until tests are added -->
+    <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
```

Comment:
Instead of skipping it is better to use a lower coverage bar, this is an example:

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/storage/azure-storage-common/pom.xml#L36